ForumProject/projects/admin.py

```diff
@@ -1,3 +1,46 @@
 from django.contrib import admin
+from .models import Project, ProjectFiles, InvestorProject, ProjectLog
 
-# Register your models here.
+class ProjectAdmin(admin.ModelAdmin):
+    list_display = ['id',
+                    'name',
+                    'startup',
+                    'description',
+                    'status',
+                    'created_at',
+                    'updated_at',
+                    'duration',
+                    'budget_currency',
+                    'budget_amount'
+                    ]
+    search_fields = ['name', 'description']
+
+admin.site.register(Project, ProjectAdmin)
+
+class ProjectFilesAdmin(admin.ModelAdmin):
+    list_display = ['id', 'project', 'file_description', 'file']
+    search_fields = ['id', 'project', 'file_description', 'file']
```

Comment:
Including `id` in `search_fields` is usually not necessary unless you specifically need to search by `id`. Users rarely search by `id` as it is not user-friendly.
Searching by the file path (`file`) may not be efficient or useful. Users generally won’t search by the file path (or name) unless you have a specific use case for it

Reply:
`id` and `file` removed from `search_fields`